lib/livebook/runtime/erl_dist/logger_gl_backend.ex

```diff
@@ -94,9 +94,12 @@ defmodule Livebook.Runtime.ErlDist.LoggerGLBackend do
   end
 
   defp log_event(level, msg, ts, md, gl, state) do
+    output = format_event(level, msg, ts, md, state)
+
     if io_proxy?(gl) do
-      output = format_event(level, msg, ts, md, state)
       async_io(gl, output)
+    else
+      send(Process.whereis(Livebook.Runtime.ErlDist.NodeManager), {:orphan_log, output})
```

Comment:
You can pass the name as an argument to `send` without looking up it's pid :) Also, to better encapsulate this, let's add a function like `NodeManager.name()`

Reply:
Actually, since it's the same as the module it's fine to inline the name, nevermind :)
```suggestion
      send(Livebook.Runtime.ErlDist.NodeManager, {:orphan_log, output})
```